mage_integrations/mage_integrations/destinations/google_cloud_storage/__init__.py

```diff
@@ -0,0 +1,93 @@
+from google.cloud.storage import Client
+from mage_integrations.destinations.base import Destination
+from mage_integrations.destinations.utils import update_record_with_internal_columns
+from datetime import datetime, timezone
+from io import BytesIO
+from typing import Dict, List
+import argparse
+import os
+import pandas as pd
+import sys
+
+
+class GoogleCloudStorage(Destination):
+    @property
+    def bucket(self) -> str:
+        return self.config['bucket']
+    
+    @property
+    def file_type(self) -> str:
+        return self.config.get('file_type')
+    
+    @property
+    def object_key_path(self) -> str:
+        return self.config['object_key_path']
+    
+    def build_client(self) -> Client:
+        return Client.from_service_account_json(
+            json_credentials_path=self.config['google_service_acc_key_filepath']
```

Comment:
Let's make sure the this key name is same with the one in the template config.json